content/en/tracing/guide/agent-obfuscation.md

```diff
@@ -0,0 +1,129 @@
+---
+title: Agent Trace Obfuscation
+kind: guide
+private: true
+disable_toc: true
+---
+
+Agent trace obfuscation is disabled by default. Enable it in your `datadog.yaml` configuration file to obfuscate all information attached to your traces.
+
+Currently this options only works with the following services:
+
+* `mongodb`
+* `elasticsearch`
+* `redis`
+* `memcached`
+* `http`
+* `remove_stack_traces`
+
+
+{{< tabs >}}
+{{% tab "MongoDB" %}}
+
+Applies to spans of type `mongodb`, more specifically: to the `mongodb.query` span metadata.
+
+```
+apm_config:
+  enabled: true
+
+  ## (...)
+
+  obfuscation:
+    mongodb:
+      enabled: true
+      keep_values:
+        - uid
+        - cat_id
+```
+
+* `keep_values` are a set of keys from which the values are not obfuscated.
```

Comment:
```suggestion
* `keep_values` - define a set of keys to exclude from agent trace obfuscation.
```
another passive voice that I think makes it unclear, so making it active for clarity.